tools/bowtie2.wdl

```diff
@@ -0,0 +1,410 @@
+            external_help: "https://bowtie-bio.sourceforge.net/bowtie2/manual.shtml#mates-can-overlap-contain-or-dovetail-each-other",
+        }
+        no_contain: {
+            description: "If one mate alignment contains the other, consider that to be non-concordant.",
+            external_help: "https://bowtie-bio.sourceforge.net/bowtie2/manual.shtml#mates-can-overlap-contain-or-dovetail-each-other",
+        }
+        no_overlap: {
+            description: "If one mate alignment overlaps the other at all, consider that to be non-concordant.",
+            external_help: "https://bowtie-bio.sourceforge.net/bowtie2/manual.shtml#mates-can-overlap-contain-or-dovetail-each-other",
+        }
+        time: "Print the wall-clock time required to load the index files and align the reads to STDERR."
+        write_unpaired_unaligned: "Write unpaired reads that didn't align to a file in gzipped FASTQ format. These reads correspond to the SAM records with the FLAGS 0x4 bit set and neither the 0x40 nor 0x80 bits set."
+        write_unpaired_aligned: "Write unpaired reads that aligned at least once to a file in gzipped FASTQ format. These reads correspond to the SAM records with the FLAGS 0x4, 0x40, and 0x80 bits unset."
+        write_paired_discordant: "Write pairs that didn't align concordantly to a file in gzipped FASTQ format. These reads correspond to the SAM records with the FLAGS 0x4 bit set and either the 0x40 or 0x80 bit set (depending on whether it's mate #1 or #2)."
+        write_paired_concordant: "Write pairs that aligned concordantly at least once to a file in gzipped FASTQ format. These reads correspond to the SAM records with the FLAGS 0x4 bit unset and either the 0x40 or 0x80 bit set (depending on whether it's mate #1 or #2)."
+        quiet: "Print nothing besides alignments and serious errors."
+        metrics_file: "Write bowtie2 alignment metrics to file. Useful for debugging."
+        metrics_stderr: "Write  bowtie2 alignment metrics to STDERR. Not mutually exclusive with `metrics_file`."
+        metrics_interval: "Report internal counters & metrics every N seconds. Only used if `metrics_file` and/or `metrics_stderr` is specified."
+        no_unal: "Suppress SAM records for unaligned reads."
+        no_head: "Suppress header lines, i.e. lines starting with @."
+        no_sq: "Suppress @SQ header lines."
+        read_group: "Read group record to include in output SAM/BAM header"
+        addl_rg_text: "add <text> ('label:value') to @RG line of SAM header"
+        omit_sec_seq: "When printing secondary alignments, Bowtie 2 by default will write out the SEQ and QUAL strings. Specifying this option causes Bowtie 2 to print an asterisk in those fields instead."
+        sam_no_quane_trunc: "Suppress standard behavior of truncating readname at first whitespace at the expense of generating non-standard SAM."
```

Comment:
Why would this be desirable?

Reply:
There are fastq files with read names that have spaces. If you've included relevant/important information after the whitespace, you'd probably want it in the BAM. That said, the solution is probably to fix your fastq files, rather than break the BAM.